src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$width: 100px;
+$height: 100px;
+$color: #eee;
+$gap: 1px;
+$tranformY: -20px;
+$padding: 10px;
+$max-width: 7 * $width + 2 * $padding + 7 * $gap;
+
+$daysAmount: 30;
+$startingDay: "fri";
+// I hope it's okay, that i've used maps and fuctions
+// instead of @each cicle. I just was really courios about
+// discovering how maps works
```

Comment:
curiosity is a nice trait
just make sure you can solve the task with `@each` too :)